README.md

```diff
@@ -1,29 +1,86 @@
 [![Build Status](https://travis-ci.com/brain-score/brain-score.web.svg?branch=master)](https://travis-ci.com/brain-score/brain-score.web)
 
-# Setup
+# Setup 
+(To run the Brain-Score Website locally)
 
-Ensure you are using `<= python@3.8`
+***NOTE: Before beginning this process, contact one of the Brain-Score Administrators listed below and request an AWS_ACCESS_KEY_ID, and an AWS_SECRET_ACCESS_KEY.  You will need these to complete this process.
```

Comment:
Isn't there still the option of using a local database configuration (sqlite)?

Reply:
Hi Martin.  I don't know.  Should this have worked automatically if I didn't have credentials?  For me, when I was trying to run the website locally, I was getting errors prior to setting up the credentials in the manner I described here.  If there are steps required to use a local database and you or someone else can point me in the right direction, then I will include those in this documentation.

Reply:
Hi Martin, I talked this over with Mike.  He said to let you know that local db support is currently on pause.  At his suggestion I removed the mention of using a local db.  So I think this should be good to go.  Also, fyi, I really only updated the part of the readme relevant to running the website locally, I didn't go through the rest (like the section about Django) in detail.

Reply:
ok I'm just worried that we'll exclude external contributions altogether if we require a connection to our database. But I guess this makes sense as supporting this option probably outweighs the benefits at the moment.

Reply:
hopefully not.  as far as i know, this only affects people who want to run the website locally and not people who want to use the brainscore repo and/or submission tutorials.  i would think this would mostly affect brainscore developers.